### PR TITLE
[BD-05][EDUCATOR-5596] Remove tinymcev5 and use the existing one from the platform

### PR DESCRIPTION
**Pull request by shimulch**
During testing, it has been discovered that the TinyMCE v5 used in the Pluggable ORA Submission editor collides with the existing TinyMCE v4 in the platform. This PR removes TinyMCE v5 and uses the version from the platform instead.

**JIRA tickets**: https://openedx.atlassian.net/browse/EDUCATOR-5596

~~**Discussions**:~~

**Dependencies**: None

**Screenshots**:
Screenshot 2021-02-26 at 1 09 32 PM

Screenshot 2021-02-26 at 1 11 29 PM

**Sandbox URL**: TBD

**Merge deadline**: None

**Testing instructions**:

1. Pull this PR into your local devstack.
2. Install `edx-ora2`.
3. Create a ORA block in studio.
4. From Edit -> Settings -> Response Editor, select WYSIWYG Editor.
5. Check that TinyMCE Response Editor loads in Studio.
6. Check that ORA Prompt editor works.
7. Publish the block.
8. Check TinyMCE loads in LMS.

**Author notes and concerns**:

TBD

**Reviewers**
- [ ] @giovannicimolin 
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @shimulch! I've created [BLENDED-778](https://openedx.atlassian.net/browse/BLENDED-778) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js [hunk 1]
```diff
@@ -50,18 +62,13 @@ tinymce.baseURL = '/xblock/resource/openassessment/static/vendors/tinymce/';
         this.elements = elements;
 
         const ctrl = this;
-
+        // this.elements.tinymce()
```
Comment (giovannicimolin) [inline openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js:65]: @shimulch Please remove this comment

File: openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
-// Tell tinymce from where it should load css, plugins etc
-tinymce.baseURL = '/xblock/resource/openassessment/static/vendors/tinymce/';
 
 (function (define) {
-  define(() => {
+  const dependencies = [];
+  const tinymceCssFile = '/static/js/vendor/tinymce/js/tinymce/skins/studio-tmce4/skin.min.css';
+
+  if (typeof window.tinymce === 'undefined') {
+    // as tinymce is not available, we need to load them
```
Comment (giovannicimolin) [inline openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js:10]:
```suggestion
    // If tinymce is not available, we need to load it
```

**Review by giovannicimolin — CHANGES_REQUESTED** [review]
Minor nits. 

Can you also remove `tinymce` from the `package.json` file and add a version bump?

File: openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js (hunk 1 above)
Comment (shimulch) [reply openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js]: Removed

File: openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js (hunk 2 above)
Comment (shimulch) [reply openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js]: Updated.

**Comment by shimulch** [comment]
@giovannicimolin, I've added content styling CSS for tinyMCE. Hopefully, it will look a bit nicer now.
Also, removed tinymce from package.json & added version bumps.

**Review by giovannicimolin — APPROVED** [review]
:+1:

- [x] I tested this:
1. Tested that the editor works on LMS and studio.
2. Checked that text is nicely displayed in the readonly version of the editor.
- [x] I read through the code
- [x] I checked for accessibility issues: fixes issues caused by previous implementation.
- [ ] ~~Includes documentation~~ NA

**Comment by shimulch** [comment]
@giovannicimolin, I've tested your commit that fixes the whitespace issue. LGTM 👍🏽 Thanks for figuring out a way to achieve that.

**Comment by jnlapierre** [comment]
Please resolve merge conflicts and we will finalize the review.

**Review by jnlapierre — APPROVED** [review]
LGTM

**Pull request merged**